src/cltk/alphabet/text_normalization.py

```diff
@@ -32,3 +33,104 @@ def remove_non_latin(input_string, also_keep=None):
     latin_chars += "".join(also_keep)
     no_latin = "".join([char for char in input_string if char in latin_chars])
     return no_latin
+
+
+def split_trailing_punct(text: str, punctuation: Optional[List[str]] = None) -> str:
+    """Some tokenizers, including that in Stanza, do not always
+    handle punctuation properly. For example, a trailing colon (``"οἶδα:"``)
+    is not split into an extra punctuation token. This function
+    does such splitting on raw text before being sent to such
+    a tokenizer.
+
+    Args:
+        text: Input text string.
+        punctuation: List of punctuation that should be split when trailing a word.
+
+    Returns:
+        Text string with trailing punctuation separated by a whitespace character.
+
+    >>> raw_text = "κατηγόρων’, οὐκ οἶδα: ἐγὼ δ᾽ οὖν"
+    >>> split_trailing_punct(text=raw_text)
+    'κατηγόρων ’, οὐκ οἶδα : ἐγὼ δ᾽ οὖν'
+    """
+    if not punctuation:
+        # What about the curly thing (``᾽``) in eg ``δ᾽``
+        punctuation = [":", "’", "”"]  # closing curly quotes
+    new_chars: List[str] = list()
+    for index, char in enumerate(text):
+        if char in punctuation:
```

Comment:
This may become 

```python
if char in punctuation and index > 0:
```
and remove 

```python
if index == 0:
    new_chars.append(char)
    continue
```